src/components/WebsiteTabs/website-tabs.scss

```
@@ -7,39 +7,45 @@
   margin-bottom: 0;
 }
 
-.website-tabs nav.bx--tabs {
+.website-tabs nav.#{$prefix}--tabs {
   min-height: 2.8rem;
 }
 
-.website-tabs ul.bx--tabs__nav {
+.website-tabs ul.#{$prefix}--tabs__nav {
   margin-left: 0;
   margin-right: 0;
 }
 
-.website-tabs li.bx--tabs__nav-item {
+.website-tabs li.#{$prefix}--tabs__nav-item {
   border: none;
   margin: 0;
   border-left: rem(1px) solid $carbon--gray-30;
 }
 
-.website-tabs li.bx--tabs__nav-item:first-child {
+.website-tabs li.#{$prefix}--tabs__nav-item:first-child {
   border-left: none;
 }
 
-.website-tabs a.bx--tabs__nav-link {
+.website-tabs a.#{$prefix}--tabs__nav-link {
+  display: block;
+  height: 100%;
   border: none;
   box-sizing: border-box;
   background: $ui-03;
   color: $text-01;
   line-height: 1.5;
   padding: $spacing-sm $spacing-md;
```

```suggestion
  padding: $spacing-sm $spacing-md;
  height: 2.5rem;
```

I think we need to set the height here too

---

puttin this in too, thanks!